zipline/testing/fixtures.py

```diff
@@ -884,9 +909,13 @@ def make_adjustment_writer_daily_bar_reader(cls):
         return cls.bcolz_daily_bar_reader
 
     @classmethod
+    def make_adjustment_db_conn_str(cls):
+        return ':memory:'
```

Comment:
There is a comment in the class docstring which includes `returns the sqlite3 connection string for the database`. Should we double that up here?